### ISSUE TITLE
[BUG] Dynamoose Get method is 50x slower than aws-sdk dynamodb getItem method

### ISSUE THREAD

**Issue by GeekEast** [dynamoose/dynamoose#1287]
### Summary:
Dynamoose get method is 50x slower than the aws-sdk `getItem` method

#### Schema
```javascript
var AWS = require('aws-sdk');
var AWSXRay = require('aws-xray-sdk');
var http = require("http")
var https = require("https")
const { model, Schema } = require('dynamoose');

AWSXRay.captureHTTPsGlobal(http)
AWSXRay.captureHTTPsGlobal(https)
console.log('start create dynamodb client', new Date().toISOString());
var ddb = AWSXRay.captureAWSClient(new AWS.DynamoDB());
console.log('complete create dynamodb client', new Date().toISOString());

var TABLE_NAME = 'TEST';

exports.handler = async (event) => {
  const schema = new Schema({
    PK: { hashKey: true, type: String },
    displayName: String,
    createdAt: Date,
    updatedAt: Date
  });

  console.log('start initialize model', new Date().toISOString());
  const model = model(TABLE_NAME, schema, {create: false});
  console.log('complete initialize model', new Date().toISOString());

  console.log('start getting item from dynamoose', new Date().toISOString());
  const res1 = await model.get('b624b2f3-feab-4b7a-a529-2c1dbd10d5e1');
  console.log('complete getting item from dynamoose', new Date().toISOString());
  console.log(res1);

  const param = {
    TableName: TABLE_NAME,
    Key: { PK: { S: 'b624b2f3-feab-4b7a-a529-2c1dbd10d5e1' } },
  };

  // redeploy trigger
  console.log('start getting item from dynamodb', new Date().toISOString());
  const res2 = await ddb.getItem(param).promise();
  console.log('complete getting item from dynamodb', new Date().toISOString());
  console.log(res2);
  return { statusCode: 200 };
};
```

### Current output and behavior (including stack trace):
![image](https://user-images.githubusercontent.com/24806887/140867105-5f86e24c-3f55-4ae7-ae63-0d739eba7420.png)

### Expected output and behavior:
Dynamoose call should have the same performance as aws-sdk, right?

### Environment:

Environment: AWS Lambda Function
Node.js version (`node -v`): `node14.x`
Dynamoose version:  `^2.8.2`

### Other information (if applicable):
- aws-xray is enabled for monitoring
- package.json
```json
{
  "name": "test-dynamodb-delay",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "aws-xray-sdk": "^3.3.3",
    "dynamoose": "^2.8.2"
  }
}
```
- lambda environment variables
```
AWS_NODEJS_CONNECTION_REUSE_ENABLED=1
```

**Comment by vasilkrastev90** [comment]
@GeekEast as a package user I was also trying to evaluate the performance as well using your code. What I noticed when running consecutive queries using both dynamo sdk client and dynamoose is that the time compares.

In your case I think the first invocation taking 426ms is probably initialising the client session. In my trials subsequent executions took less time.

The second one taking 8ms seems to be the dynamoose get item

And the 3rd one  taking 51ms seems to be the native client.

Below is a trace with only 2 sdk native client invocations:

![image](https://user-images.githubusercontent.com/8515347/140927743-bfee4f24-4791-4713-b77e-9cd05d4d1192.png)


And a trace of 2 sdk invocations and 2 dynamoose invocations:

![image](https://user-images.githubusercontent.com/8515347/140927837-2266d789-5cb9-498a-bc02-a866ead66ed1.png)

What I notice is that only the first dynamo invocation takes longer, which suggests that this is a session initialization by the client.

**Comment by fishcharlie** [comment]
Closing as duplicate of #1260.

TLDR: fixed in v3 alpha.
